client/graphviz.js

```diff
@@ -83,34 +89,41 @@
         } else if (ir.match(/^[A-Z]/)) {
           console.log('eval',ir)
 
-          if (ir.match(/^DOT (strict )?digraph/)) {
+          if (ir.match(/^DOT (strict )?(di)?graph/)) {
             context.graph = ir.replace(/DOT /,'')
-          }
+          } else
 
           if (ir.match(/^LINKS/)) {
             let text = context.want.map(p=>p.text).join("\n")
             let links = (text.match(/\[\[.*?\]\]/g)||[]).map(l => l.slice(2,-2))
             let tree = nest()
             links.map((link) => {
-              if (ir.match(/^LINKS HERE -> NODE/)) {
-                dot.push(`${quote(context.name)} -> ${quote(link)}`)
-              }
+              if (ir.match(/^LINKS HERE (->|--) NODE/)) {
+                let kind = ir.match(/->|--/)
+                dot.push(`${quote(context.name)} ${kind} ${quote(link)}`)
+              } else
               if (ir.match(/^LINKS NODE -> HERE/)) {
```

Comment:
digraph only?

Reply:
Here I correctly capture `kind` and substitute it into the dot format instruction. This is the case I tested.